### PR TITLE
chore: ungate multi-column keys from the feature flag

### PR DESCRIPTION
**Pull request by agavra**
### Description 

lets gooooo (docs + minor changes to remove the feature flag)

### Testing done 

Added corresponding historical tests and updated some of the other tests

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/concepts/schemas.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ key and value of the underlying {{ site.ak }} topic message.
 A column is defined by a combination of its [name](#valid-identifiers), its [SQL data type](#sql-data-types),
 and possibly a namespace.
 
-Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively.
-ksqlDB supports a single key column only.
+Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively while value
```
Comment (JimGalasyn) [inline docs/concepts/schemas.md:20]:
```suggestion
Key columns for a stream have a `KEY` suffix. Key columns for a table have a `PRIMARY KEY` suffix. Value 

```

File: docs/concepts/schemas.md [hunk 2]
```diff
@@ -17,17 +17,17 @@ key and value of the underlying {{ site.ak }} topic message.
 A column is defined by a combination of its [name](#valid-identifiers), its [SQL data type](#sql-data-types),
 and possibly a namespace.
 
-Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively.
-ksqlDB supports a single key column only.
+Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively while value 
+columns have no namespace suffix. There can be multiple columns in either namespace, given that
```
Comment (JimGalasyn) [inline docs/concepts/schemas.md:21]:
```suggestion
columns have no namespace suffix. There can be multiple columns in either namespace, if

```

File: docs/concepts/schemas.md [hunk 3]
```diff
@@ -17,17 +17,17 @@ key and value of the underlying {{ site.ak }} topic message.
 A column is defined by a combination of its [name](#valid-identifiers), its [SQL data type](#sql-data-types),
 and possibly a namespace.
 
-Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively.
-ksqlDB supports a single key column only.
+Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively while value 
+columns have no namespace suffix. There can be multiple columns in either namespace, given that
+the underlying serialization format supports it (the `KAFKA` format, for example, does not).
```
Comment (JimGalasyn) [inline docs/concepts/schemas.md:22]:
```suggestion
the underlying serialization format supports it.

!!! note 
    The `KAFKA` format doesn't support multi-column keys.





```

File: docs/concepts/schemas.md [hunk 4]
```diff
@@ -17,17 +17,17 @@ key and value of the underlying {{ site.ak }} topic message.
...
-ksqlDB supports a single key column only.
+Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively while value 
+columns have no namespace suffix. There can be multiple columns in either namespace, given that
+the underlying serialization format supports it (the `KAFKA` format, for example, does not).
 
-Value columns have no namespace suffix. There can be one or more value columns.
-
-For example, the following example statement declares a stream with a single
-key column and several value columns:
+For example, the following example statement declares a stream with several
```
Comment (JimGalasyn) [inline docs/concepts/schemas.md:24]:
```suggestion
For example, the following statement declares a stream with multiple

```

File: docs/developer-guide/serialization.md [hunk 5]
```diff
@@ -429,10 +439,14 @@ Single field (un)wrapping
 
 ### (de)serialization of single keys
 
-At this time, ksqlDB supports only a single key column and that key columns
-must be unwrapped, i.e. not contained within a outer record or object. See
-the next two sections on single values for more information about wrapped
-and unwrapped data.
+ksqlDB assumes that any single key is unwrapped, i.e. not contained within an outer
```
Comment (JimGalasyn) [inline docs/developer-guide/serialization.md:442]:
```suggestion
ksqlDB assumes that any single key is unwrapped, which mean that it's not contained in an outer
```

File: docs/developer-guide/serialization.md [hunk 6]
```diff
@@ -429,10 +439,14 @@ Single field (un)wrapping
...
+record or object. Conversely, ksqlDB assumes that any key with multiple columns
+(e.g. `CREATE STREAM K1 INT KEY, K2 INT KEY, C1 INT`) is wrapped, i.e is a record
```
Comment (JimGalasyn) [inline docs/developer-guide/serialization.md:444]:
```suggestion
(for example, `CREATE STREAM K1 INT KEY, K2 INT KEY, C1 INT`) _is_ wrapped, which means that it is a record
```

File: docs/developer-guide/serialization.md [hunk 7]
```diff
@@ -429,10 +439,14 @@ Single field (un)wrapping
...
+with each column as a field within the key. 
+
+To declare a single column key that is wrapped, you may specify a `STRUCT` type
```
Comment (JimGalasyn) [inline docs/developer-guide/serialization.md:447]:
```suggestion
To declare a single-column key that's wrapped, specify a `STRUCT` type
```

File: docs/developer-guide/serialization.md [hunk 8]
```diff
@@ -429,10 +439,14 @@ Single field (un)wrapping
...
+with a single column (e.g. `K STRUCT<F1 INT> KEY`). See the next two sections
```
Comment (JimGalasyn) [inline docs/developer-guide/serialization.md:448]:
```suggestion
with a single column. for example, `K STRUCT<F1 INT> KEY`. See the next two sections 
```

**Review by JimGalasyn — APPROVED** [review]
LGTM, with a few suggestions.

File: docs/concepts/schemas.md [hunk 9]
```diff
@@ -106,9 +111,10 @@ Avro-formatted data. If you want to use Protobuf- or JSON-formatted data,
 substitute `PROTOBUF`, `JSON` or `JSON_SR` for `AVRO` in each statement.
 
 !!! note
-    ksqlDB handles the `JSON` and `JSON_SR` formats differently. While the
-    `JSON` format is capable of _reading_ the schema from {{ site.sr }},
-    `JSON_SR` both reads and registers new schemas, as necessary.
+    ksqlDB handles the `JSON` and `JSON_SR` formats differently. `JSON_SR` reads and 
+    registers new schemas with {{ site.sr }}, as necessary while `JSON` requires you
```
Comment (vcrfxia) [inline docs/concepts/schemas.md:115]:
```suggestion
    registers new schemas with {{ site.sr }} as necessary, while `JSON` requires you
```

File: docs/developer-guide/serialization.md [hunk 10]
```diff
@@ -127,11 +129,11 @@ split into columns.
 For example, given a SQL statement such as:
 
 ```sql
-CREATE STREAM x (ID BIGINT, NAME STRING, AGE INT) WITH (VALUE_FORMAT='DELIMITED', ...);
+CREATE STREAM x (ORGID BIGINT KEY, ID BIGINT KEY, NAME STRING, AGE INT) WITH (FORMAT='DELIMITED', ...);
 ```
 
-ksqlDB splits a value of `120, bob, 49` into the three fields with `ID` of
-`120`, `NAME` of `bob` and `AGE` of `49`.
+ksqlDB splits a key of `120, 21` and a value of `bob, 49` into the four fields (two keys and two values)
```
Comment (vcrfxia) [inline docs/developer-guide/serialization.md:135]: Is it valid to have spaces here? I thought the default delimiter was a comma (with no space)?

**Review by vcrfxia — APPROVED** [review]
Hooray! 🎉 Thanks @agavra !

File: docs/developer-guide/serialization.md (hunk 10 above)
Comment (agavra) [reply docs/developer-guide/serialization.md:135]: I think you are right!

**Pull request merged**